interop/executor.go

```diff
@@ -183,6 +223,12 @@ func (e *Executor) GetTxStatus(ctx context.Context, hash common.Hash, dbTx pgx.T
 		return
 	}
 
+	c, werr := e.meter.Int64Counter("get_tx_status")
+	if err != nil {
+		e.logger.Warnf("failed to create check_tx counter: %s", werr)
+	}
+	c.Add(context.Background(), 1)
```

Comment:
They are not exactly the same, on RPC they always increment, here only increments in case it succeed, I can change naming, but also the context is different, see https://github.com/0xPolygon/agglayer/pull/94/files#diff-5684f26b2a20f741cdfeaadbaef67cdd4c6d22a7a09c2f7fd392bdd8531463d7R24

Reply:
I like the different status approach though